### PR TITLE
Alert Title in Alert Tables + minor changes on AlertDetails & AlertTables

### LINKED ISSUE #615: Show Alert Title in Alert Tables
### Description

This ticket makes sure to add the `Alert Title` in all the alert tables found throughout the application. Alert Title is a prominent thing and should be added as the first column in all related tables

### Acceptance Criteria

In the RuleDetails page, the alerts should:

- Add alert title to schema (`AlertSummary` and `AlertDetails`)
- Add the alert title as the first column in the table found in RuleDetails page
- Add the alert title as first column in the ListAlerts page
- Show the dedup string in the table found in RuleDetails page

### PR DESCRIPTION
**Pull request by alexmylonas**
## Background

Currently, on `Alerts Table` we show `AlertId` which is not that helpful. We need to change this, so we can show `Alert Title `which it is generated by the backend and provides context to each `Alert`. 

Also, we deemed helpful to change columns positions for the same Table and shorten the AlertId to be more readable. Added more fields on embedded AlertTable on RuleDetails page

Check screenshots for visual context!

Closes #615
## Changes

- `Alert Title` on Alert Table
- Shorten the display of `AlertId` to the first 7 chars in `AlertsTable` and in `Breadcrumb`.
- Rearrange columns in Alerts Table proposed order -> `Title`,`Created at`, `Severity`, short `AlertId`,`Events Count`,`Last Matched At`
- Added `Title`, `Last Matched At` in `AlertDetails`
- Added more fields in embedded Alert Table on `RuleDetails` (see screenshot)
- Updated `Fragments` for `Alerts`

## Testing

- Locally
- Deployed Env

## Screenshots
Screenshot 2020-04-15 at 3 12 30 PM
Screenshot 2020-04-15 at 3 13 11 PM
Screenshot 2020-04-15 at 3 13 54 PM

### CODE REVIEW

File: web/src/components/Breadcrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -41,6 +41,8 @@ const widthSentinelStyles = {
   textOverflow: 'ellipsis' as const,
 };
 
+const transformBreadcrumbText = item => (isHash(item.text) ? shortenId(item.text) : item.text);
+
```
Comment (alexmylonas) [inline web/src/components/Breadcrumbs/Breadcrumbs.tsx:45]: @3nvi how do you feel about this? Do you have any alternative approach?

**Comment by kostaspap** [comment]
> Other than that, I'm ok if the table columns are the ones the log analysis team wanted. @kostaspap can you verify?

I have already reviewed the code, yup I verified they are correct.

File: web/src/constants.ts [hunk 2]
```diff
@@ -31,6 +31,8 @@ export const INCLUDE_UPPERCASE_REGEX = new RegExp('(?=.*[A-Z])');
 
 export const INCLUDE_SPECIAL_CHAR_REGEX = new RegExp('[^\\d\\sA-Za-z]');
 
+export const IS_HASH_REGEX = new RegExp('[a-f0-9]{32}');
```
[the remark this answers is not in the document]
Comment (alexmylonas) [reply web/src/constants.ts:35]: you are right, updated it!

File: web/src/components/Breadcrumbs/Breadcrumbs.tsx (hunk 1 above)
Comment (3nvi) [reply web/src/components/Breadcrumbs/Breadcrumbs.tsx:26]: we  can optionally just inline `transformBreadcrumbText`. Not a big deal tho

File: web/src/pages/AlertDetails/AlertDetailsInfo/AlertDetailsInfo.tsx [hunk 3]
```diff
@@ -45,7 +45,15 @@ const AlertDetailsInfo: React.FC<AlertDetailsInfoProps> = ({ alert, rule }) => {
...
-                ID
+                TITLE
+              </Label>
+              <Text size="medium" color="black">
+                {alert.title}
+              </Text>
+            </Box>
+            <Box my={1}>
+              <Label mb={1} is="div" size="small" color="grey300">
+                FULL ALERT ID
```
[the remark this answers is not in the document]
Comment (3nvi) [reply web/src/pages/AlertDetails/AlertDetailsInfo/AlertDetailsInfo.tsx:56]: I'm still not 100% sure, but  I'll go with whatever you feel

**Pull request merged**